src/components/Modal/Modal.js

```diff
@@ -123,26 +124,15 @@ const overlayClassName = {
  * Global body styles
  */
 
-/* eslint-disable no-unused-expressions */
+/* eslint-disable-next-line no-unused-expressions */
 injectGlobal`
    /* Remove scroll on the body when react-modal is open */
-  .ReactModal__Body--open {
+  .ReactModal__Html--open {
     height: 100%;
-    overflow: hidden;
+    overflow-y: hidden;
```

Comment:
btw, the `no-scroll` lib also adds `overflow: hidden` to the document. Not sure if it will work, but have you tried removing this altogether. Will simplify things a lot! :)

Reply:
well, we only run `no-scroll` for iOS devices. So I think we need to keep this for android for instance